admin/tests-integration/sql.service.spec.js

```diff
@@ -1,35 +1,46 @@
 'use strict'
-/* global describe beforeEach beforeAll afterEach it expect jasmine spyOn fail */
+/* global describe beforeEach beforeAll afterEach it xit expect jasmine spyOn fail */
 
 require('dotenv').config()
 const sql = require('../services/data-access/sql.service')
 const sqlPool = require('../services/data-access/sql.pool.service')
+const TYPES = require('tedious').TYPES
 
 describe('sql.service:integration', () => {
-  beforeAll(async (done) => {
+  beforeAll(() => {
     sqlPool.init()
-    done()
   })
 
-  it('should permit select query with no parameters', async (done) => {
+  it('should permit select query with no parameters', async () => {
+    let settingsRows = await sql.query('SELECT * FROM Settings')
+    expect(settingsRows).toBeDefined()
+    expect(settingsRows.length).toBe(1)
```

Comment:
@jon-shipley @GeorgeChatzigiannis if the expectation fails (for example when `settingsRows.length` equals 2, it hangs until Jasmine times out.

I get the predicted warning: `(node:45871) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.` but nothing else until the jasmine time out.

What am I missing/overlooking here?